src/main/java/org/pmiops/workbench/model/Cohort.java

```diff
@@ -0,0 +1,25 @@
+package org.pmiops.workbench.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Cohort {
```

Comment:
While I like protos generally, I'm not sure about their use here.

There are two types of objects our APIs will expose: those stored in FireCloud and those stored in our database.

The former need to be stored in FC as JSON; the latter probably will be written to Cloud SQL using Hibernate (with different fields as different columns in tables.) In both cases, they would be returned to clients as JSON, for maximum portability.

You *could* have an internal representation of objects as protos used neither for persistence nor for returning to clients, but that doesn't seem worthwhile. (If we were returning the objects or storing them as protos, it would make more sense.)

I will definitely look into Swagger and codegen. There's also this Spring option which looks interesting:

https://projects.spring.io/spring-restdocs/

Either way I think representing objects as POJOs fits in better with these frameworks.